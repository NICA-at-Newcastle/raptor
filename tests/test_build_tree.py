"""Tree builder tests."""

import pytest


@pytest.mark.slow
def test_builds_tree(tree_builder, cinderella_text, memory_storage, snapshot):
    """Test that raptor builds the tree from the cinderella text."""

    chunks = cinderella_text.split(".")

    _ = tree_builder.build_from_chunks(chunks)

    assert snapshot == memory_storage.to_dict()
