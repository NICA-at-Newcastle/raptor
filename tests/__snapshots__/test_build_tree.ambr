# serializer version: 1
# name: test_builds_tree
  dict({
    'child_to_parent': dict({
    }),
    'index_to_node': dict({
    }),
    'layers': dict({
    }),
    'parent_to_children': dict({
    }),
  })
# ---
